### DISCUSSION TITLE
Welcome to outline-client Discussions!

### DISCUSSION THREAD

**Discussion by daniellacosse** [OutlineFoundation/outline-apps#1283] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ironmann250** [comment]
hi,
i tried changing the name "Outline" to a different one but as much as i change the code, the name remains the same, is there a way to change the title name?
